beater/config/config.go

```diff
@@ -92,6 +100,57 @@ type Config struct {
 	DefaultServiceEnvironment string                  `config:"default_service_environment"`
 
 	Pipeline string
+
+	// TODO: wat is with this naming
```

Comment:
I think it's OK now, and we can remove this? Maybe not ideal, perhaps we can come back to it later

Reply:
yeah, definitely not a comment that should make it into master :)